.github/workflows/E2Etests.yml

```diff
@@ -0,0 +1,71 @@
+name: End-to-end tests
+on: [push]
+
+jobs:
+  cypress-e2e-headless:
+    name: Cypress e2e
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ["14"]
+        browser: ["firefox", "chrome"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup and Run backend
+        run: |
+          git clone --branch Added_test_docker-compose https://github.com/CSCfi/beacon-network
+          cd beacon-network
+          docker build -t cscfi/beacon-network .
+          docker-compose -f docker-compose-test.yml up -d
+          sleep 30
+      - name: Add api_keys to registry
+        run: |
+          cat ./testApiKeys.sql | docker exec -i beacon-network_db_registry_1 psql -U user -d registry
+      - name: Add beacon to registry
+        run: |
+          curl -X 'POST' \
+            'localhost:8080/services' \
+          -H 'Authorization: 07b4e8ed58a6f97897b03843474c8cc981d154ffe45b10ef88a9f127b15c5c56' \
+          -H 'accept: application/json' \
+          -H 'Content-Type: application/json' \
+          -d '{
+          "type": "beacon",
+          "url": "https://staging-elixirbeacon.rahtiapp.fi/service-info"
```

Comment:
did not notice this before - this should not be a real url, or better yet why do we have that real url there?

Reply:
I think i mentioned this to you before, but the idea was to use the dev beacon as a placeholder and add both the beacon and beaconV2 mocks at the same time when the beaconV2 is implemented. But i can make the mock now if you want it.

Reply:
ok, then we can leave it for now and treat the mocks as part of another PR.